src/htcp.cc

```diff
@@ -313,6 +314,19 @@ htcpHexdump(const char *tag, const char *s, int sz)
 #endif
 }
 
+static bool
+parseUint16(char *buf, int sz, uint16_t &out, const char *field)
```

Comment:
This parsing function does not modify `buf`.  The rest is just `const` application to parameter values themselves:

```suggestion
parseUint16(const char * const buf, const int sz, uint16_t &out, const char * const field)
```

I would rename `field` to `fieldDescription` or simply `description`, but I do not insist on that.